.github/workflows/test.yaml

```diff
@@ -0,0 +1,62 @@
+name: test
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        required: true
+        type: string
+      date:
+        required: true
+        type: string
+      sha:
+        required: true
+        type: string
+
+jobs:
+  conda-cpp-tests:
+    secrets: inherit
+    uses: rapidsai/shared-workflows/.github/workflows/conda-cpp-tests.yaml@branch-24.12
+    with:
+      build_type: nightly
+      branch: ${{ inputs.branch }}
+      date: ${{ inputs.date }}
+      sha: ${{ inputs.sha }}
+  conda-python-tests:
+    secrets: inherit
+    uses: rapidsai/shared-workflows/.github/workflows/conda-python-tests.yaml@branch-24.12
+    with:
+      build_type: nightly
+      branch: ${{ inputs.branch }}
+      date: ${{ inputs.date }}
+      sha: ${{ inputs.sha }}
+  wheel-tests-pylibwholegraph:
+    secrets: inherit
+    uses: rapidsai/shared-workflows/.github/workflows/wheels-test.yaml@branch-24.12
+    with:
+      build_type: nightly
+      branch: ${{ inputs.branch }}
+      date: ${{ inputs.date }}
+      sha: ${{ inputs.sha }}
+      script: ci/test_wheel_pylibwholegraph.sh
+      matrix_filter: map(select(.ARCH == "amd64"))
```

Comment:
Just checking - do we know why all these packages skip ARM tests? Should we comment about that?

Reply:
I have no idea.

@BradReesWork added that for `wholegraph` over a year ago: https://github.com/rapidsai/wholegraph/pull/24

@tingyu66 added that constraint for `cugraph-dgl` and `cugraph-pyg` about 9 months ago: https://github.com/rapidsai/cugraph/pull/4133

Maybe they or @alexbarghi-nv could share why it's like that.

Reply:
There are no packages available on arm64 for pytorch.

Reply:
I tried re-enabling those tests and they all failed again because it pulled the pytorch CPU package.

Reply:
As far as I know, most ARM64 users of pytorch are going through DLFW or some other means.

Reply:
alright thank you. I do see `aarch64` CUDA builds of `pytorch` 2.4.1 (the latest version) at https://anaconda.org/conda-forge/pytorch/files, so maybe in the future there's an opportunity to turn these tests back on.

I'll leave it as-is for the purpose of this PR yet, to get more CI up and running so we can move development over here from the `cugraph` repo.